src/components/events/EventSidebar.tsx

```diff
@@ -0,0 +1,291 @@
+              <PopoverPortal>
+                {filterState.filter === 'pick' && (
+                  <DatePopover
+                    range={filterState.date}
+                    setRange={(val) => {
+                      setFilterState((old) => {
+                        return {
+                          filter: 'pick',
+                          clubs: old.clubs,
+                          date: val,
+                          order: old.order,
+                          types: old.types,
+                        };
+                      });
+                    }}
+                  />
+                )}
+              </PopoverPortal>
+            </Popover>
+          </RadioGroupItem>
+        </RadioGroup>
+      </div>
+      <div className="flex flex-col space-y-7.5">
+        <h2 className="text-sm font-bold text-slate-500">Search for Club</h2>
+        <EventClubSearchBar
+          addClub={(val) => {
+            setFilterState((old) => {
+              if (!old.clubs.includes(val)) {
+                return {
+                  filter: old.filter,
+                  clubs: [...old.clubs, val],
+                  order: old.order,
+                  types: old.types,
+                };
+              }
+              return old;
+            });
+          }}
+        />
+        <div className="space-y-2 p-1">
+          {filterState.clubs.map((value) => (
+            <SelectedClub
+              key={value}
+              clubId={value}
+              setFilterState={setFilterState}
+            />
+          ))}
+        </div>
+      </div>
+      <div className="flex flex-col space-y-7.5">
+        <h2 className="text-sm font-bold text-slate-500">Order</h2>
+        <RadioGroup
+          className="flex flex-col space-y-2.5"
+          value={filterState.order}
+          onValueChange={(value) => {
+            setFilterState((old) => {
+              return {
+                filter: old.filter,
+                clubs: old.clubs,
+                order: value as (typeof order)[number],
+                types: old.types,
+              };
+            });
+          }}
+        >
+          {order.map((value) => (
+            <RadioGroupItem
+              key={value}
+              value={value}
+              className="h-10 w-64 rounded-lg bg-white py-2.5 drop-shadow-sm"
+            >
+              <div className="ml-4 mr-10 flex h-min flex-row items-center space-x-2.5 text-center">
+                <div className="flex h-5 w-5 items-center justify-center rounded-xl border-2 border-gray-50 p-0.5 shadow-sm ">
+                  <RadioGroupIndicator className="h-2.5 w-2.5 rounded-full bg-blue-primary" />
+                </div>
+                <p className="w-full text-xs font-extrabold text-slate-500">
+                  {value}
+                </p>
+              </div>
+            </RadioGroupItem>
+          ))}
+        </RadioGroup>
+      </div>
+      <div className="flex hidden flex-col space-y-7.5">
```

Comment:
These two conflict, for now remove flex but add a comment saying this is for future implementation

Reply:
removed flex and added comment